packages/docusaurus-plugin-client-redirects/src/createRedirectPageContent.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+type CreateRedirectPageOptions = {
+  toUrl: string;
+};
+
+export default function createRedirectPageContent({
+  toUrl,
+}: CreateRedirectPageOptions) {
+  return `
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8">
+       <meta http-equiv="refresh" content="0; url=${toUrl}">
+    <script>
+      const redirectLink = '${toUrl}' + location.search + location.hash;
+      document.write('<link rel="canonical" href="' + redirectLink + '">');
+      document.write('<title>Redirecting to ' + redirectLink + '</title>');
+      document.write('</head>')
+      document.write('<body>')
+      document.write('If you are not redirected automatically, follow this <a href="' + redirectLink + '">link</a>.')
+      document.write('</body>')
+      setTimeout(() => {
+        window.location.assign(redirectLink)
```

Comment:
@lex111 this part is inspired by the plugin you wrote here: https://github.com/single-spa/single-spa.js.org/blob/master/website/src/plugins/docusaurus-plugin-redirects/src/index.js

Was wondering why do you need a setTimeout?

I've seen a gatsby plugin that does not have this timeout

https://github.com/dhakerShiv/gatsby-plugin-client-side-redirect/blob/master/clientSideRedirect.js

Reply:
Oh yes, this is an  old-fashion version of browser redirect. We had better remove `setTimeout` and `document.write` calls.

Reply:
done